08-rails/active-record-models.md

```diff
@@ -5,137 +5,133 @@
+Rails provides a REPL (similar to irb and pry) that pre-loads all the application information when started. It's one of the most useful tools I've encountered. Let's spend some time exploring it. From your application root, run `$ rails console`:
 
 ```bash
- ❤️  :: rails console
+$ rails console
 Running via Spring preloader in process 48975
 Loading development environment (Rails 4.2.6)
 2.3.0 :001 >
 ```
 
-Now we're in the _console_, from here we can do all the stuff that irb and pry provide, plus have access to all of our Rails models. Use the console to create an __instance__ of our Student _model_:
+Now we're in the _console_, from here we can do all the stuff that irb and pry provide, plus have access to all of our Rails models. Use the console to create an __instance__ of our Book _model_:
 
 ```ruby
-2.3.0 :001 > s = Student.new
- => #<Student id: nil, name: nil, pie: nil, motto: nil, created_at: nil, updated_at: nil>
-2.3.0 :002 > s.attributes
- => {"id"=>nil, "name"=>nil, "pie"=>nil, "motto"=>nil, "created_at"=>nil, "updated_at"=>nil}
-2.3.0 :003 >
+2.3.0 :001 > b = Book.new
+ => #<Book id: nil, title: nil, author: nil, description: nil, created_at: nil, updated_at: nil>
 ```
 
 We can assign data to the instance using plain Ruby:
 ```ruby
-2.3.0 :003 > s.name = "Libby"
- => "Libby"
-2.3.0 :004 > s.pie = "Apple"
- => "Apple"
-2.3.0 :006 > s.attributes
- => {"id"=>nil, "name"=>"Libby", "pie"=>"Apple", "motto"=>nil, "created_at"=>nil, "updated_at"=>nil}
+2.3.0 :002 > b.title = "Between The World and Me"
+ => "Between The World and Me"
+2.3.0 :003 > b.author = "Ta-Nehisi Coates"
+ => "Ta-Nehisi Coates"
+2.3.0 :004 > b
+ => #<Book id: nil, title: "Between The World and Me", author: "Ta-Nehisi Coates", description: nil, created_at: nil, updated_at: nil>
 ```
 
 When we are happy with the data in our _model_, we can write it to the database using a single command:
 
 ```ruby
-2.3.0 :007 > s.save
-   (0.3ms)  begin transaction
-  SQL (1.8ms)  INSERT INTO "students" ("name", "pie", "created_at", "updated_at") VALUES (?, ?, ?, ?)  [["name", "Libby"], ["pie", "Apple"], ["created_at", "2016-04-17 18:17:52.649965"], ["updated_at", "2016-04-17 18:17:52.649965"]]
+2.3.0 :005 > b.save
+(5.9ms)  begin transaction
+SQL (2.2ms)  INSERT INTO "books" ("title", "author", "created_at", "updated_at") VALUES (?, ?, ?, ?)  [["title", "Between The World and Me"], ["author", "Ta-Nehisi Coates"], ["created_at", 2017-03-17 05:33:02 UTC], ["updated_at", 2017-03-17 05:33:02 UTC]]
    (1.5ms)  commit transaction
  => true
-2.3.0 :008 >
 ```
 
 __Question: ZOMG what just happened?__
 The Active Record DSL translates our Ruby code into SQL, and then executes it.
 
 ```ruby
-Student.where(pie: "apple").order('birthday DESC')
+Book.where(author: "Ta-Nehisi Coates")
 ```
 becomes
 
 ```sql
-SELECT * FROM "students" WHERE "pie" = "apple"  ORDER BY birthday DESC;
+SELECT * FROM "students" WHERE "author" = "Ta-Nehisi Coates";
```

Comment:
Will we go over enough SQL in [this lecture](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/07-databases/01-what-is-a-database.md) for this example to be meaningful to the students?